views/pages/index.ejs

```diff
@@ -4,16 +4,18 @@
     <meta charset="utf-8">
     <title><%= Math.round(currently.temperature) %>° <%= currently.summary.charAt(0).toUpperCase() + currently.summary.slice(1).toLowerCase() %> <%= params.formatted_location %> Weather</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Why remove `shrink-to-fit=no` ?

Reply:
Lighthouse (progressive webapp analyzer - https://github.com/GoogleChrome/lighthouse) complains about unrecognized properties on viewport meta tag